tests/runtests.py

```diff
@@ -232,9 +232,6 @@ def setup_collect_tests(start_at, start_after, test_labels=None):
     # tests.
     log_config['loggers']['django']['level'] = 'ERROR'
     settings.LOGGING = log_config
-    settings.SILENCED_SYSTEM_CHECKS = [
-        'fields.W342',  # ForeignKey(unique=True) -> OneToOneField
-    ]
```

Comment:
Can we drop this change, using `silence_checks` with `override_settings` just on specific tests? (We're not removing the current approach no?) — It leads to a lot of noise on the PR. 🤔